modules/renter/pool.go

```diff
@@ -0,0 +1,101 @@
+package renter
+
+import (
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/modules/renter/contractor"
+)
+
+// A hostPool is a collection of hosts that can store data. New hosts are
+// drawn from a HostDB, and contracts are negotiated with them on demand.
+type hostPool struct {
```

Comment:
seems like the way you are using the host pool vs. the way it is documented have diverged. As the architecture has evolved, the host pool now serves a different purpose